src/components/Header.vue

```diff
@@ -14,15 +14,15 @@ div.aw-navbar
         // If only a single view (the default) is available
         b-nav-item(v-if="activityViewsDaily.length === 1", v-for="view in activityViewsDaily", :key="view.name", :to="view.pathUrl + '/' + view.hostname")
           div.px-2.px-lg-1
-            icon(name="clock")
-            | Activity
+            icon(name="calendar-day")
+            | Today
```

Comment:
It really isn't "Today" though as you can select any day.

I think "Activity" was better. In the future I only want there to be an "Activity" button merged with the summary.

Reply:
I thought about that, but "Activity" is too generic, and the other option I considered was "Daily" which isn't really good either. If you click the button, it takes you to the "today view", from which you can later switch date. Good enough for me, and imo an improvement. 

When/if we merge them into a single view it might make sense to go back to "Activity", but in the meanwhile, I think this is an improvement.